admin/cincoctrl/cincoctrl/users/decorators.py

```diff
@@ -3,8 +3,8 @@
 
 def user_has_repo_access(func):
```

Comment:
I think this decorator is no longer used as this PR removes all references to it in preference for the mixin. Is that right? Worth removing?